Providers/PythonProvider.hpp

```diff
@@ -164,7 +166,6 @@ PythonProvider::PythonProvider (
     T const& name)
     : m_Name (name)
     , m_FD (PythonProvider::INVALID_SOCKET)
-    , m_pid (-2)
```

Comment:
>d ( [](start = 10, length = 3)

Seems like we need to either revert this change or initialize m_pid to something else.

Reply:
changing back to -2.